galleon-pack/src/main/resources/layers/standalone/microprofile/layer-spec.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" ?>
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2020, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="microprofile">
+    <dependencies>
+        <layer name="microprofile-config" optional="true"/>
+        <layer name="microprofile-fault-tolerance" optional="true"/>
+        <layer name="microprofile-jwt" optional="true"/>
+        <layer name="microprofile-metrics" optional="true"/>
+        <layer name="microprofile-openapi" optional="true"/>
+        <layer name="open-tracing" optional="true"/>
```

Comment:
@rhusar Just out of curiosity, why we don't include microprofile-health?

Reply:
@yersan Fixed, thanks. The reason being that 'microprofile-health' layer did not exist in January.